SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Reporting Security Issues
+
+To report a security issue, please email
+[example@email.com](mailto:example@email.com)
```

Comment:
I think we can list myself (alex.gaynor@gmail.com) and Cory (lukasaoz@gmail.com).